frontend/package.json

```diff
@@ -66,15 +66,15 @@
     "zone.js": "~0.13.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^16.2.11",
+    "@angular-devkit/build-angular": "^16.2.12",
```

Comment:
This patch fixes "dependabot" issue [212](https://github.com/havochvatten/MSP-Symphony/security/dependabot/212)